I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 87, in script
    python:
  File "game/script.rpy", line 89, in <module>
    pc.move(choice)
  File "game/script.rpy", line 19, in move
    if location in self.neighbors:
AttributeError: 'Player' object has no attribute 'neighbors'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 87, in script
    python:
  File "/Applications/renpy-6.99.12.3-sdk/renpy/ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-6.99.12.3-sdk/renpy/python.py", line 1715, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 89, in <module>
    pc.move(choice)
  File "game/script.rpy", line 19, in move
    if location in self.neighbors:
AttributeError: 'Player' object has no attribute 'neighbors'

Darwin-16.7.0-x86_64-i386-64bit
Ren'Py 6.99.12.3.2123
Azule 1.0
